app/models/dog_searcher.rb

```diff
@@ -38,7 +38,7 @@ def search
         @dogs = Dog.where("dogs.name ILIKE ?", "%#{@params[:q]}%")
       end
     else
-      @dogs = Dog.where("status IN (?)", PUBLIC_STATUSES)
+      @dogs = Dog.includes(:photos, :foster, :primary_breed, :secondary_breed).where("status IN (?)", PUBLIC_STATUSES)
```

Comment:
is this the only spot you need this?  might make sense to change all paths here

you always want @dogs so maybe at line 21 you do

```
@dogs = Dog.includes(:photos, :foster, :primary_breed, :secondary_breed)
```

and then where `@dogs` is now you do `@dogs = @dogs.where(whatever the clause is now)`